doc/conf.py

```diff
@@ -269,6 +256,39 @@
 # so a file named "default.css" will overwrite the builtin "default.css".
 html_static_path = ["_static"]
 
+# redirects = {}
+
+# configuration for sphinxext.opengraph
+ogp_site_url = "https://xarray.pydata.org/en/latest/"
+ogp_image = "https://xarray.pydata.org/en/stable/_static/dataset-diagram-logo.png"
+ogp_custom_meta_tags = [
+    '<meta name="twitter:card" content="summary_large_image" />',
+    '<meta property="twitter:site" content="@xarray_dev />',
+    '<meta name="image" property="og:image" content="https://xarray.pydata.org/en/stable/_static/dataset-diagram-logo.png">',
+]
+
+# Redirects for pages that were moved to new locations
+# Enable this once there's a workaround for the following warning
+# WARNING: the sphinx_reredirects extension does not declare if it is safe for parallel reading, assuming it isn't - please ask the extension author to check and make it explicit
+
+# redirects = {
+#     "terminology": "user-guide/terminology.html",
+#     "data-structures.html": "user-guide/data-structures.html",
+#     "indexing.html": "user-guide/indexing.html",
+#     "interpolation.html": "user-guide/interpolation.html",
+#     "computation.html": "user-guide/computation.html",
+#     "groupby.html": "user-guide/groupby.html",
+#     "reshaping.html": "user-guide/reshaping.html",
+#     "combining.html": "user-guide/combining.html",
+#     "time-series.html": "user-guide/time-series.html",
+#     "weather-climate.html": "user-guide/weather-climate.html",
+#     "pandas.html": "user-guide/pandas.html",
+#     "io.html": "user-guide/io.html",
+#     "dask.html": "user-guide/dask.html",
+#     "plotting.html": "user-guide/plotting.html",
+#     "duckarrays.html": "user-guide/duckarrays.html",
+# }
+
```

Comment:
This is a list of pages that were moved to new locations. It'd be nice/convenient to set up redirects for these so as to avoid `404 errors`. I was going to use the `sphinx_reredirects` ext until I noticed that sphinx emits the following warning which results into build failures on ReadtheDocs:

```bash
WARNING: the sphinx_reredirects extension does not declare if it is safe for parallel reading, assuming it isn't - please ask the extension author to check and make it explicit
```


One remedy is to set these redirects from readthedocs settings page: https://docs.readthedocs.io/en/stable/user-defined-redirects.html

Reply:
I made you a maintainer on RTD so you should be able to play around with that.

Did you try `sphinxext-rediraffe`? From a quick look this should do the same as `sphinx_reredirects`, but it seems more mature and does declare itself as safe for parallel read / write.

Reply:
> Did you try sphinxext-rediraffe? From a quick look this should do the same as sphinx_reredirects, but it seems more mature and does declare itself as safe for parallel read / write.

I just tried `sphinxext-rediraffe`, and it works like a charm. Thank you for the suggestion